### PR TITLE
Add a paragraph about slashing in runtime disputes section from the guide

### PR DESCRIPTION
**Pull request by tdimitrov**
Follow up for https://github.com/paritytech/polkadot/pull/6345

### CODE REVIEW

File: roadmap/implementers-guide/src/runtime/disputes.md [hunk 1]
```diff
@@ -123,4 +123,17 @@ applied:
...
+   also a spam precaution. A legitimate client will send only confirmed disputes to the runtime.
+
+# Rewards and slashing
+
+After the disputes are filtered the validators participating in the disputes are rewarded and more importantly the
+offenders are slashed. Generally there can be two types of punishments:
+* "against valid" - the offender claimed that a valid candidate is invalid.
+* "for invalid" - the offender claimed that an invalid candidate is valid.
+
+Validators participating in inconclusive disputes are not slashed. Thanks to the applied filtering (described in the
```
Comment (sandreim) [inline roadmap/implementers-guide/src/runtime/disputes.md:135]: nit: I would also detail what `inconclusive` means in this case .

**Review by sandreim — APPROVED** [review]

File: roadmap/implementers-guide/src/runtime/disputes.md (hunk 1 above)
Comment (tdimitrov) [reply roadmap/implementers-guide/src/runtime/disputes.md:135]: Yes, this is a good suiggestion.

**Review by s0me0ne-unkn0wn — APPROVED** [review]

File: roadmap/implementers-guide/src/runtime/disputes.md [hunk 2]
```diff
@@ -123,4 +123,19 @@ applied:
...
-   also a spam precaution. A legitimate client will send only confirmed disputes to the runtime.
\ No newline at end of file
+   also a spam precaution. A legitimate client will send only confirmed disputes to the runtime.
+
+# Rewards and slashing
+
+After the disputes are filtered the validators participating in the disputes are rewarded and more importantly the
+offenders are slashed. Generally there can be two types of punishments:
+* "against valid" - the offender claimed that a valid candidate is invalid.
+* "for invalid" - the offender claimed that an invalid candidate is valid.
```
Comment (ordian) [inline roadmap/implementers-guide/src/runtime/disputes.md:133]: Not sure it's worth mentioning such details at the moment. We might end up not using slashing for "for invalid" disputes and use "negative era points" instead. We'll probably have a full page for slashing eventually once the design is finalized. Also this section is named "Rewards and slashing", but no mention of rewards here.

Comment (sandreim) [reply roadmap/implementers-guide/src/runtime/disputes.md:133]: I am in favor of keeping details for now as it shows current design and obsolete this once we have the design ironed out and that new page reflecting it.

File: roadmap/implementers-guide/src/runtime/disputes.md [hunk 3]
```diff
@@ -123,4 +123,19 @@ applied:
...
+
+A dispute might be inconclusive. This means that it has timed out without being confirmed. A confirmed dispute is one
+containing votes more than the byzantine threshold (1/3 of the active validators). Validators participating in
+inconclusive disputes are not slashed. Thanks to the applied filtering (described in the previous section) one can be
+confident that there are no spam disputes in the runtime. So if a validator is not voting it is due to another reason
+(e.g. being under DoS attack). There is no reason to punish such validators with a slash.
```
Comment (eskimor) [inline roadmap/implementers-guide/src/runtime/disputes.md:139]: Thank you!

**Review by eskimor — APPROVED** [review]

File: roadmap/implementers-guide/src/runtime/disputes.md (hunk 2 above)
Comment (eskimor) [reply roadmap/implementers-guide/src/runtime/disputes.md:133]: Agreed, we can adjust anytime.

**Review by mrcnski — COMMENTED** [review]
I'm slightly too late, but approved! Thanks for updating the guide!

**Pull request merged**